content/blueprints/spec-intrinsic-functions.md

```diff
@@ -270,14 +270,91 @@ outputs:
+
+If however, the *referenced node* shares a [scaling group]({{< relref "blueprints/scaling.md" >}}#scaling-policy-and-scaling-groups-configuration) with the *referencing node*, the ambiguity may be resolved.
+
+The actual details are a little more intricate, and what follows is an explanation followed by an example.
+
+First, the term *referencing node* depends on where in the blueprint, `get_attribute` is used. If it is used in a node operation's inputs (e.g. `cloudify.interfaces.lifecycle.start`), *referencing node* is the node template under which the operation is defined.
+
+If on the other hand, `get_attribute` is used in a relationship operation's inputs (e.g. `cloudify.interfaces.relationship_lifecycle.establish`), *referencing node* is actually *referencing nodes*, which are the source and target nodes involved in the relationship operation. As we'll see, both can be used as a *referencing node*, and the first of them to resolve the ambiguity will be used.
+
+Resolving the ambiguity for `get_attribute` usages in the blueprint `outputs` is not supported.
+
+Moving on.
+
+So, let `A` be the *referencing node* and `B` be the *referenced node*. If `A` and `B` belong to some scaling group and that scaling group's instances contain only one instance of `B`, `get_attribute` will resolve to using that `B`'s instance when evaluating the `get_attribute`.
+
+And now, an example:
+
+{{< gsHighlight  yaml  >}}
+node_templates:
+  db_server:
+    type: cloudify.nodes.DBMS
+  web_server:
+    type: cloudify.nodes.WebServer
+    interfaces:
+      cloudify.interfaces.lifecycle
+        configure:
+          implementation: some_plugin.tasks.configure
+          inputs:
+            # here, the referencing node is the web_server and the referenced
+            # node is the db_server
+            db_connection_url: { get_attribute: [db_server, connection_url] }
+    relationships:
+      - target: db_server
+        type: cloudify.relationships.connected_to
+        source_interfaces:
+          cloudify.interfaces.relationship_lifecycle:
+            preconfigure:
+              implementation: some_plugin.tasks.my_preconfigure
+              inputs:
+                # here the referencing nodes are web_server and db_server and the
+                # referenced node is db_server (i.e. a node can reference itself)
+                db_connection_url: { get_attribute: [db_server, connection_url] }
+
+groups:
+  db_and_webserver:
+    members: [db_server, web_server]
+
+policies:
+  scaling_policy1:
+    type: cloudify.policies.scaling
+    properties:
+      default_instances: 2
+    targets: [db_and_webserver]
+{{< /gsHighlight >}}
+
+The previous blueprint defines an application with one scaling group `db_and_webserver` that has 2 instances (initially). Each group instance contains
```

Comment:
not sure you need '(initially)' here. Unless you intend to scale it in this example.. it is more confusing. I understand (I think) you're trying to say that this can later be scaled.. but I think it's irrelevant here.